### PR TITLE
Empy state view

### PR DESCRIPTION
**Pull request by josetheis-stackit**

### CODE REVIEW

File: src/features/channels/store/channel.slice.ts [hunk 1]
```diff
@@ -34,6 +36,9 @@ const slice = createSlice({
   name: 'channel',
   initialState,
   reducers: {
+    selectChannel: (state: ChannelState, { payload }) => {
```
Comment (mmiranda-stackit) [inline src/features/channels/store/channel.slice.ts:39]: Please add tests to this case

File: src/pages/chat-page/ChatPage.tsx [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
-    </HeaderSection>
-    <MainSection>
-      <ChatContent />
-    </MainSection>
-    <FooterSection>
-      <ChatFooter />
-    </FooterSection>
-  </>
-);
+const ChatPage = () => {
```
Comment (mmiranda-stackit) [inline src/pages/chat-page/ChatPage.tsx:10]: I suggest to use the router here. You can create a new route for empty state, and upon channel click you can navigate to the chat route passing an id, which will load your channel content.

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]
left some comments

File: src/pages/chat-page/ChatPage.tsx (hunk 2 above)
Comment (josetheis-stackit) [reply src/pages/chat-page/ChatPage.tsx]: Done

**Review by mmiranda-stackit — APPROVED** [review]
there is still pending to add unit tests

**Pull request merged**